include/aws/testing/aws_test_harness.h

```diff
@@ -478,7 +481,8 @@ static inline int s_aws_run_test_case(struct aws_test_harness *harness) {
         const size_t leaked_bytes = aws_mem_tracer_bytes(allocator);
         if (leaked_bytes) {
             aws_mem_tracer_dump(allocator);
-            PRINT_FAIL_INTERNAL0("Test leaked memory: %zu bytes %zu allocations", leaked_bytes, leaked_allocations);
+            PRINT_FAIL_WITHOUT_LOCATION(
```

Comment:
It always annoyed me how the final "test leaked" and "test failed" messages printed their file location and line no. It made it look like the aws_test_runner.h code is what had bugs